README.md

```diff
@@ -525,6 +526,34 @@ Using the private key:
 })();
 ```
 
+#### Getting the IDs of the Issuers of a signature
```

Comment:
To be honest, while I appreciate you writing some documentation for this, I'm not sure if this operation is common enough to warrant a full example in the top-level README. Once this PR is merged and included in a release, the method will automatically be included in https://openpgpjs.org/openpgpjs/Signature.html (generated from the JSDoc), I feel like that should be enough.